src/main/java/com/fvv/bookstore/dao/OrderDAO.java

```diff
@@ -20,4 +25,37 @@
 	 * @throws DaoException when a problem in database happens.
 	 */
 	void addOrder(final Order order) throws DaoException;
+	
+	/**
+	 * Lists all the order by search month.
```

Comment:
look up for all javadocs containing this "the" with this context, and remove all of them

Reply:
ok